internal/network/manifests_generator.go

```diff
@@ -68,6 +69,81 @@ spec:
   osImageURL: ""
 `
 
+const snoDnsmasqConf = `
+address=/{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/{{.HOST_IP}}
+addn-hosts=/etc/api-int.host
+`
+
+const apiIntHosts = `
+{{.HOST_IP}} api-int api-int.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}
+`
+
+const forceDnsDispatcherScript = `
+export IP="{{.HOST_IP}}"
```

Comment:
any chance to add logging here?

Reply:
before create, yes sure. it is not closed code